services/motion/builtin/builtin.go

```diff
@@ -275,11 +275,22 @@ func (ms *builtIn) MoveOnGlobe(
 	if !ok {
 		return false, fmt.Errorf("cannot move base of type %T because it is not a Base", baseComponent)
 	}
+	positionOnly := false
+	profile, ok := extra["motion_profile"]
```

Comment:
[nit] we should check that extra != nil so there is no nil pointer deference error